openpype/hosts/maya/plugins/publish/extract_maya_scene_raw.py

```diff
@@ -58,16 +58,12 @@ def process(self, instance):
         else:
             members = instance[:]
 
-        loaded_containers = None
-        if set(self.add_for_families).intersection(
-                set(instance.data.get("families")),
-                set(instance.data.get("family").lower())):
-            loaded_containers = self._add_loaded_containers(members)
-
         selection = members
-        if loaded_containers:
-            self.log.info(loaded_containers)
-            selection += loaded_containers
+        if set(self.add_for_families).intersection(
+                set(instance.data.get("families"))) or \
+                instance.data.get("family") in self.add_for_families:
+            selection += self._add_loaded_containers(members)
+            self.log.info(selection)
```

Comment:
I'd recommend refraining from logging all nodes. It tends to be a very large list, thus still very hard to debug even if it's there and it slows down current interfaces as well with that many nodes. Unlike before where it was just the loaded containers that were logged.

Reply:
Agreed